### PR TITLE
Markdown files clean-up

### PR DESCRIPTION
**Pull request by VutonDesign**
A simple pull request that fixes mistakes in a couple of markdown files.

## Things fixed
 - "javasript" changed to "JavaScript"
- Added a word to make the sentence sound better
- "in" changed to "on"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ A pure-web version or Chrome-app version is planned to follow at some point in t
 Platform | Status
 -------- | -------------
 iOS      | Released in App Store
```
Comment (thisandagain) [inline README.md]: This should probably be consistent.

Comment (VutonDesign) [reply README.md]: @thisandagain Okay, Will we keep it as "in" or "on"?

Comment (thisandagain) [reply README.md]: "in" seems more idiomatic so I think I'd keep this as is.

Comment (VutonDesign) [reply README.md]: Right, I'll change that back. 👍

**Pull request merged**